packages/ui-extensions/src/surfaces/admin/extension-targets.ts

```diff
@@ -5,13 +5,31 @@ import type {
   StandardApi,
   CustomerSegmentationTemplateApi,
   ActionExtensionApi,
+  ProductDetailsConfigurationApi,
+  ProductVariantDetailsConfigurationApi,
 } from './api';
 import {AnyComponentBuilder} from '../../shared';
 
 type CustomerSegmentationTemplateComponent = AnyComponentBuilder<
   Pick<Components, 'CustomerSegmentationTemplate'>
 >;
 
+type ProductConfigurationComponents = AnyComponentBuilder<
+  Pick<
+    Components,
```

Comment:
In this slack [thread](https://shopify.slack.com/archives/C04K7BZDH3N/p1689690591439869) we discussed also adding the Resource List component and Thumbnail. Should this be added here?